BrushedMotorControllerFirmware/Kconfigs/Kconfig.position_cntrl

```diff
@@ -1,20 +1,30 @@
 menu "Position Control Configuration"
 
+config POS_CONTROL_ENABLE
```

Comment:
if there are multiple configs in the same config, you can use:
```
if POS_CONTROL_ENABLE

config AAA
    bool "AAA description"
    default y


config BBB
    int "BBB description"
    default 123


endif
```

Reply:
corrected